api/tests/unit/application/account-recovery/account-recovery-controller_test.js

```diff
@@ -38,6 +39,27 @@ describe('Unit | Controller | account-recovery-controller', () => {
       expect(response.statusCode).to.equal(204);
     });
 
+    describe('#checkAccountRecoveryDemand', () => {
+
+      it('should return serialized user', async () => {
```

Comment:
**question**
Je ne vois pas de rapport entre 
- le titre du test 'should return serialized user'
- les assertions

```
expect(usecases.getUserByAccountRecoveryDemand).to.have.been.calledWith({ temporaryKey });
expect(userSerializer.serialize).to.have.been.calledWith({ userId: 1234 });
```